samples/matter/common/src/bridge/ble_connectivity_manager.h

```diff
@@ -245,6 +245,7 @@ class BLEConnectivityManager {
 	static void DiscoveryNotFound(bt_conn *conn, void *context);
 	static void DiscoveryError(bt_conn *conn, int err, void *context);
 	static int StartGattDiscovery(bt_conn *conn, BLEBridgedDeviceProvider *provider);
+	static bool ParamChangeRequestHandler(struct bt_conn *conn, struct bt_le_conn_param *param);
```

Comment:
Declaration should be put under `ifdef CONFIG_BRIDGE_SET_BT_CONNECTION_PARAMS` as it is not used in other case.